app/views/shared/_device_card.html.erb

```diff
@@ -13,6 +13,9 @@
         <p class="lead"><code><%= device.serial -%></code></p>
 
         <% if size == :full %>
+        <% if device.plugin %>
+        <%= render("shared/#{device.device_type.downcase}_plugin_card", device: device) rescue nil -%>
```

Comment:
Might be better calling it something like `_type_plugin_partial` rather than `card`